src/coreComponents/linearAlgebra/interfaces/hypre/HypreMGR.hpp

```diff
@@ -84,21 +84,23 @@ class MGRStrategyBase
   HYPRE_Int m_numLabels[numLevels]{ -1 };              ///< Number of dof labels kept
   HYPRE_Int * m_ptrLabels[numLevels]{ nullptr };       ///< Pointers to each level's labels, as consumed by MGR
 
-  MGRFRelaxationMethod m_levelFRelaxMethod[numLevels];    ///< F-relaxation strategy for each level (single or multilevel)
-  MGRFRelaxationType m_levelFRelaxType[numLevels];        ///< F-relaxation type for each level
-  MGRInterpolationType m_levelInterpType[numLevels];      ///< Interpolation type for each level
-  MGRRestrictionType m_levelRestrictType[numLevels];      ///< Restriction type for each level
-  MGRCoarseGridMethod m_levelCoarseGridMethod[numLevels]; ///< Coarse grid method for each level
-  HYPRE_Int m_levelSmoothType[numLevels]{ -1 };           ///< Smoother type for each level
-  HYPRE_Int m_levelSmoothIters[numLevels]{ -1 };          ///< Number of smoother iterations for each level
+  MGRFRelaxationType m_levelFRelaxType[numLevels];           ///< F-relaxation type for each level
+  HYPRE_Int m_levelFRelaxIters[numLevels]{ -1 };             ///< Number of F-relaxation iterations for each level
+  MGRInterpolationType m_levelInterpType[numLevels];         ///< Interpolation type for each level
+  MGRRestrictionType m_levelRestrictType[numLevels];         ///< Restriction type for each level
+  MGRCoarseGridMethod m_levelCoarseGridMethod[numLevels];    ///< Coarse grid method for each level
+  MGRGlobalSmootherType m_levelGlobalSmootherType[numLevels]; ///< Global smoother type for each level
+  HYPRE_Int m_levelGlobalSmootherIters[numLevels]{ -1 };     ///< Number of global smoother iterations for each level
 
-  HYPRE_Int m_numRestrictSweeps{ -1 }; ///< Number of restrict sweeps
-  HYPRE_Int m_numInterpSweeps{ -1 };   ///< Number of interpolation sweeps
+  // *** Do we need to expose these? They are not used at the moment ***
```

Comment:
This would be for post-smoothing of the interpolation operators. It's currently commented out in MGR's code because of some past issues with the implementation. The hypre team needs to revisit it at some point. So, it's fine to leave these options commented out for now at least